boilerplate/process.json

```diff
@@ -0,0 +1,13 @@
+{
+  "apps" : [{
+    "name"        : "api",
+    "script"      : "app.js",
+    "watch"       : false,
+    "env": {
+      "NODE_ENV": "development"
```

Comment:
Imo also worth mentioning in the readme how you can run the system in dev vs production mode.